engines/imagemagick/Dockerfile

```diff
@@ -22,9 +27,13 @@ COPY target/${env.project_artifactId}-${env.project_version}.jar /usr/bin
 
 RUN ln /usr/bin/${env.project_artifactId}-${env.project_version}.jar /usr/bin/${env.project_artifactId}.jar && \
     yum install -y $IMAGEMAGICK_DEP_RPM_URL && \
-    yum install -y $IMAGEMAGICK_LIB_RPM_URL $IMAGEMAGICK_RPM_URL && \
+    if [ "$(uname -m)" = "x86_64" ]; then \
+            yum install -y $IMAGEMAGICK_LIB_RPM_URL_AMD64 $IMAGEMAGICK_RPM_URL_AMD64; \
+        else \
+            yum install -y $IMAGEMAGICK_LIB_RPM_URL_ARM64 $IMAGEMAGICK_RPM_URL_ARM64; \
+        fi && \
     yum clean all && \
-    rpm -e --nodeps libgs
+    rpm -qa | grep libgs && rpm -e --nodeps libgs || echo "libgs package not found"
```

Comment:
Is the `|| echo "libgs package not found"` a wanted/desired change? Is it acceptable for the Docker image to be built and pushed anyways (aka, is the Docker image working?) even if libgs is not found? Why has this logic been added?

Reply:
"old" version of amd64 imagemagick from github install also libgs - which was after that step uninstalled (probably for license reason), new version of arm64 imagemagick don't have that package - that was the reason to add that logic here. But anyway according to Giovanni suggestion I change amd64 url to also use last one from nexus - in that case both version doesn't install libgs, so I delete that step at all.